README.md

```diff
@@ -251,9 +251,7 @@ modules should be required in the subprocess collecting coverage:
 
 ## Caching
 
-You can run `nyc` with the optional `--cache` flag, to prevent it from
-instrumenting the same files multiple times. This can significantly
-improve runtime performance.
+`nyc`'s default behavior is to cache instrumented files to disk to prevent instrumenting source files multiple times, and speed `nyc` execution times. You can disable this behavior by running `nyc` with the `--cache false` flag. You can also change the default cache directory from `./node_modules/.cache/nyc` by setting the `--cacheDir` flag.
```

Comment:
Isn't it `--cache-dir`?

Reply:
Turns out... that while this is processed with the rest of the config, it's not actually specified in the `yargs` -- which I assume means won't come through. Assumedly it can be specified from `.nycrc` files though.
https://github.com/istanbuljs/nyc/blob/master/lib/config-util.js#L105-L111

1. I can add this to the `yargs` settings.
2. I can remove the reference to it and let someone else work on this later.

Reply:
It would be great to see it added! Cc @bcoe